pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -376,3 +395,15 @@ func (r *ReconcileContainerJFR) createObjectIfNotExists(ctx context.Context, ns
 	logger.Info("Already exists")
 	return nil
 }
+
+func equals(a, b []corev1.VolumeMount) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i, val := range a {
+		if val != b[i] {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
Let's use `cmp.Equal` instead of this function. It has some extra handling for pointer fields in structs, of which there's one in VolumeMount. It's also a transitive dependency of ours already, so we're not adding anything new.

Reply:
Ah my bad, didn't know we had that!

Reply:
It's new to me too :smile: